RTech.CoffeeMachine.Api.Tests/BrewCoffee/Controllers/CoffeeMachineControllerTests.cs

```diff
@@ -95,17 +89,16 @@ public async Task GetBrewStatus_WhenEndpointCalled_ShouldReturnCorrectStatusCode
     public async Task GetBrewStatus_WhenApril1st_ShouldReturnEmpty418(long numRequestsSoFar)
     {
         var cache = new MemoryCache(new MemoryCacheOptions());
-        cache.Set(CoffeeMachineController.CACHE_KEY_NUM_REQUESTS, numRequestsSoFar);
+        cache.Set(BrewStatusUnavailableFilter.CACHE_KEY_NUM_REQUESTS, numRequestsSoFar);
```

Comment:
The april fool's teapot logic, was refactored into an ActionFilter (see `BrewStatusUnavailableFilter.cs`)